src/Controller/InstallProjectController.php

```diff
@@ -110,6 +110,10 @@ public function configureAction(Request $request)
 
         if ($inputData->has('configuration/php_cli')) {
             $tensideConfig->set('php_cli', $inputData->get('configuration/php_cli'));
+        } elseif ('' !== PHP_BINARY && file_exists(PHP_BINARY)) {
```

Comment:
That would be a question for @discordier 
This fixed my issue of using the wrong PHP binary which resulted in multiple issues (e.g. timezone not set, intl extension missing etc.)